modules/administration/pages/monitoring.adoc

```diff
@@ -288,9 +290,13 @@ image::monitoring_configure_formula.png[scaledwidth=80%]
 Monitoring formulas can also be configured for System Groups, by applying the same configuration used for individual systems inside the corresponding group.
 ====
 
+// The WebUI says that we describe the available arguments in the exporters formula, but I don't see it. Might be best to put it in the Salt Guide, then xref from here. --LKB 2020-08-11
```

Comment:
The help string says: _Please refer to the documentation for available options._. These are different for every exporter. I'm not sure if that's in scope of product documentation. @renner , could you comment on this please?

Reply:
Let's address this in https://github.com/SUSE/spacewalk/issues/12001 so we can merge this one now.